### DISCUSSION TITLE
Cannot get wildcard to save

### DISCUSSION THREAD

**Discussion by gravelfreeman** [sereneblue/chameleon#600] [Q&A]
Not sure how to do this. Basically I want to wildcard my own domains instead of creating a rule for each k8s service.

![image](https://github.com/user-attachments/assets/98528aaa-eae0-4e8d-8246-a21968336521)

**Comment by sereneblue** [comment]
Hi @gravelfreeman,

Chameleon does not support wildcard syntax. Please reference the wiki: https://sereneblue.github.io/chameleon/wiki/whitelist/

**Reply by gravelfreeman** [reply]
> Hi @gravelfreeman,
> 
> Chameleon does not support wildcard syntax. Please reference the wiki: https://sereneblue.github.io/chameleon/wiki/whitelist/

I added a feature request! #601

**Comment by gravelfreeman** [comment]
This has been a bummer for me since I started using Chameleon.

I just saw regex patterns. Is there a way to tell Chameleon with regex to allow all sub domains or example

`examples.domain.com`

How would I setup the regex so that it allows all `examples` subdomain to be whitelisted?

**Reply by sereneblue** [reply]
Sorry for the late response.

If you're trying to use a whitelist profile for a specific subdomain, you can specify `examples.domain.com` as the URL in the whitelist editor. This can get a little tricky, if you have multiple rules that target the same domain. The rule used will be the first found match in the  order of how the profiles are listed.

Regex only kicks in once there is a domain match.
